### DISCUSSION TITLE
Project Integration and Onboarding

### DISCUSSION THREAD

**Discussion by alexanderniebuhr** [astrolicious/astrolicious#2] [Board]
# Project Submission and Evaluation Process

To ensure the **_Astrolicious_** community organization remains dynamic and maintains high-quality projects, we will implement a structured submission and evaluation process for integrating new projects. This process is designed to be transparent, fair, and conducive to selecting projects that will contribute positively to the ecosystem.

### Submission Process
Prospective projects will be invited to submit their applications through an anonymous submission form. This system is designed to gather essential metrics about each project automatically. These metrics will form the basis of an initial score, assessing the project's potential impact, quality, and alignment with the community's goals.

### Evaluation and Voting
Once the submission is finished, the collected metrics and project details will be shared with the community members inside the organization. They will have the opportunity to vote and express their interest in the projects. This democratic approach allows the community to have a voice in which projects they believe will add the most value.

Following the community voting, a board comprising experienced members of the **_Astrolicious_** ecosystem will convene to review the project. They will analyze the score, votes, and expressed interest to make informed decisions on which project to integrate. This board will ensure that the final selection aligns with the community's strategic vision and the **_Astrolicious_** ecosystem's overall health.

### Maintenance, Archiving, and Removal
An integral part of this process is the ongoing assessment of project vitality. Projects that become unmaintained over time will be subject to review. Depending on the situation, such projects may be archived, moved out of the organization, or clearly marked as unmaintained. This ensures that the community organization remains a reliable and up-to-date resource for users and contributors. Specific criteria and processes for these decisions will be detailed further in this discussion thread.

# Project Integration Options

Projects wishing to join the **_Astrolicious_** community organization have two primary paths to consider:

## Integrated Projects
Integrated projects will be fully embraced within the community organization. Integration is highly encouraged and comes in two forms:

- **Monorepository Integration:** Ideal for smaller projects, allowing for centralized maintenance, issue tracking, and contributions.
- **Independent Repository Integration:** Suited for larger projects that require their own repository.

Both forms of integration offer equal levels of benefits, ensuring that project size or structure does not affect the potential for success within the community.

## Separated Projects
Projects that opt to remain separated from the community organization's will still be recognized under the **_Astrolicious_** organization name, benefiting from increased visibility and marketing. However, these projects will not be eligible for the full range of benefits provided to integrated projects. It is crucial for separated projects to maintain active development and adhere to community standards, as failure to do so may result in removal from the organization.

# Benefits for Integration

Integrated projects will gain access to a variety of benefits designed to support growth, collaboration, and visibility:

- **Funding and Bounties:** Opportunities for financial support to drive project development and reward contributions.
- **Maintainer Pool:** Access to a shared pool of maintainers to help with project upkeep and innovation.
- **Known Package Scope:** Projects will be part of a recognized scope, enhancing discoverability and trust within the ecosystem.
- **Marketing and Support:** Enhanced visibility through the community's marketing efforts, along with support from fellow developers and maintainers.
- **Collaborative Opportunities:** Participation in a shared maintenance group, fostering a collaborative environment for tackling common challenges.

# Requirements for Integration

To ensure a cohesive and efficient ecosystem, integrated projects are required to:

- Adopt a standardized template and structure for consistency across the community.
- Unify tooling as much as possible to simplify collaboration and contribution efforts.
- Participate actively in the shared maintenance group, contributing to the collective success of the community.

Exceptions to these requirements may be considered on an individual basis, ensuring flexibility for unique project needs.

**Comment by florian-lefebvre** [comment]
I wonder if we really need a dedicated platform with a website! tbh a few channels with some bots on discord would maybe less work for the same result in the end! I don't know!
